commands/run.go

```diff
@@ -101,13 +102,28 @@ func newRunCmd() *cobra.Command {
 
 			model := args[0]
 			prompt := ""
-			if len(args) == 1 {
-				if debug {
-					cmd.Printf("Running model %s\n", model)
-				}
-			} else {
+			args_len := len(args)
+			if args_len != 1 {
```

Comment:
```suggestion
			if args_len > 1 {
```
nit, but I think it better expressed the intent.